desktop/lib.go

```diff
@@ -76,6 +74,24 @@ var issueMap = map[string]string{
 	"Other":                       "9",
 }
 
+/*func initLantern() *app.App {
+		flags         := flashlight.ParseFlags()
```

Comment:
@atavism Can we remove this commented code?

Reply:
Done